index/miner/meta.go

```diff
@@ -0,0 +1,163 @@
+package miner
```

Comment:
This new file is an extraction of code from `miners.go` which contained all the code for the miners index. 
The `meta.go` file has all the code related to metadata gathered of miners, such as location, user-agent or any off-chain data that might exist in the future.

Not only is now in a different file, but the data was separated from on-chain data too. It's now also mantained in a separate goroutine. This was done since it didn't make sense to tie off-chain metadata scrapping with on-chain data. On-chain data needs to be refreshed when the chain has a new Tipset. Off-chain data might have its own triggers that might be completely different than the chain.

TL;DR: more separation and structure.